src/main/java/org.jitsi.meet.test/mobile/ToolbarView.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright @ Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.mobile;
+
+import io.appium.java_client.*;
+import io.appium.java_client.pagefactory.*;
+
+import org.jitsi.meet.test.mobile.base.*;
+
+import org.openqa.selenium.*;
+
+/**
+ * Page object for the mobile toolbar area.
+ */
+public class ToolbarView extends AbstractMobilePage
```

Comment:
Don't we call it toolbox? Also, how about getting a reference to the toolbox, and then geting the hangup button out of it? That would introduce the machinery we need to click other buttons in the future. WDYT?

Reply:
You mean a reference to the toolbox container element ? I'm not sure if that makes sense with how the things are currently being located...

Reply:
And I can rename it to toolbox - sure

Reply:
Yes, I meant the container. Then we'd find the buttons inside the container.

Reply:
It will not be easier this way. You'll have to add accessibility ID to the container and then each button will still need to have something that identifies it, so that we can find it - I guess that would be accessibility Id, so...

Reply:
That sounds good to me though :-) But as I saiid, I think you are better qualified to make a judgement here.